org.sparkleshare.SparkleShare.json

```diff
@@ -0,0 +1,136 @@
+{
+    "app-id": "org.sparkleshare.Sparkleshare",
+    "branch": "master",
+    "command": "sparkleshare",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "metadata": "metadata",
+    "separate-locales": false,
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=system-bus",
+        "--socket=session-bus",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        "--share=ipc",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "mono",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://download.mono-project.com/sources/mono/mono-4.2.2.30.tar.bz2",
```

Comment:
This is a very old release?

Reply:
From last year. I'll update to the latest 4.x series (from a few months back, but newer than that in most distros) as the bindings don't compile on the 5.x series yet and I don't have the knowledge right now to fix it.